### PR TITLE
Feat/ Environment deployments Table

### PR DESCRIPTION
**Pull request by WilliamTraoreee**
# What does this PR do?

[Link to JIRA Ticket](https://qovery.atlassian.net/browse/FRT-341)

- Add Environment deployment tab
- Add Environment deployment table

image

### Global

- [x] This PR does not introduce any breaking change
- [ ] This PR introduces breaking change(s) and has been labeled as such
- [x] I have found someone to review this PR and pinged him

### Store

- [x] This PR introduces new store changes

### NX

- [x] I have run the dep-graph locally and made sure the tree was clean i.e no circular dependencies
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`

### Clean Code

- [x] I made sure the code is type safe (no any)
- [ ] I have included a feature flag on my feature, if applicable

### CODE REVIEW

**Comment by RemiBonnet** [comment]
A preview environment was automatically created via Qovery.  
Click on the link below to follow its deployment and use it.  
:point_right: [[PR] staging - Feat/ Environment deployments Table - 2022-06-08T15:38:05Z](https://console.qovery.com/platform/organization/141c07c8-0dd9-4623-983b-3fdd61867255/projects/977bd069-20f7-4b24-bd95-28b44174cf43/environments/a2d9dcda-ef57-4f65-923a-5021587e8b8b/applications)

File: libs/shared/ui/src/lib/styles/main.scss [hunk 1]
```diff
@@ -26,6 +26,7 @@ body {
   overflow-x: hidden;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  @apply bg-element-light-lighter-400;
```
Comment (bdebon) [inline libs/shared/ui/src/lib/styles/main.scss:29]: Wow we missed that the body of the website was not white ??

Comment (bdebon) [reply libs/shared/ui/src/lib/styles/main.scss:29]: Why not using the new way we found for tailwind apply ? 
`background-color: theme(blablalb)`

Comment (WilliamTraoreee) [reply libs/shared/ui/src/lib/styles/main.scss:29]: Oh yes ! I change that :)

**Review by bdebon — COMMENTED** [review]
Hello @WilliamTraoreee !
Code is clean and integration is nice.
There are some weird stuff though.

Can we avoid showing all over again the placeholder when we already have the data in stored in the env?
https://user-images.githubusercontent.com/6163954/172857272-379a0035-bab9-4f6f-ad15-96f185505b77.mp4


The filter status does not work for me. Neither the ordering.
https://user-images.githubusercontent.com/6163954/172857497-d652918a-d755-4dd1-b1f4-80293e14fec6.mp4

This looks everything except cool...
CleanShot 2022-06-09 at 15 18 02@2x

Do you think you can handle these problems?

**Pull request merged**